Xamarin.Forms.Core/PlatformConfiguration/iOSSpecific/MenuItem.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace Xamarin.Forms.PlatformConfiguration.iOSSpecific
+{
+	using FormsElement = Forms.MenuItem;
+
+	public static class MenuItem
+	{
+		public static readonly BindableProperty ContextActionDisplayProperty = BindableProperty.Create(nameof(ContextActionDisplay), typeof(ContextActionDisplay), typeof(MenuItem), ContextActionDisplay.Text);
```

Comment:
`typeof(MenuItem)` - this is likely referring to this static class; this should probably be `typeof(Forms.MenuItem)` or `typeof(FormsElement)`.